### ISSUE TITLE
Run tarsnap jobs in a batch

### ISSUE THREAD

**Issue by igalic** [Brainsware/puppet-tarsnap#6]
rather than trying to splay tarsnap jobs in cron in such a way that they don't conflict, it would be nice to be able to run them serially.

This could be done in a class with almost the same parameters as `tarsnap::periodic` has, which then takes (an array? of) hashes to configure the locations.

```yaml
locations:
  - web:
      - /foo/bar
      - /baz/zomg
  - mail:
      - /foo/baz
      - /baz/quux
```
we would create a *single* shell script (`/usr/local/bin/tarsnap-batch which optionally cleans up, too) to be run from cron.

This class can optionally be included in init.pp through a parameter (`enable_batch`?).

Thoughts?

**Comment by fraenki** [comment]
I like the idea. Especially with limited bandwith it is crucial to run them serially. :+1:

**Comment by igalic** [comment]
fixed with #8!
